.gitignore

```diff
@@ -2,3 +2,4 @@ coverage.out
 *~
 coverage.txt
 profile.out
+.idea
```

Comment:
Please, do not add further entries in gitignore. You can use a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for this.